### PR TITLE
kube_codegen: expose applyconfig-openapi-schema flag for client generatio

### PR DESCRIPTION
**Pull request by soltysh**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
This PR exposes `--applyconfig-openapi-schema` flag which allows overriding the openapi-schema in applyconfiguration-gen.

#### Special notes for your reviewer:
/assign @thockin 
/cc @JoelSpeed 

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/code-generator/kube_codegen.sh [hunk 1]
```diff
@@ -594,6 +599,7 @@ function kube::codegen::gen_client() {
             --output-dir "${out_dir}/${applyconfig_subdir}" \
             --output-pkg "${applyconfig_pkg}" \
             --external-applyconfigurations "${applyconfig_external}" \
+            --openapi-schema "${applyconfig_openapi_schema}" \
```
Comment (thockin) [inline staging/src/k8s.io/code-generator/kube_codegen.sh:602]: is passing `--openapi-schema ""` valid ?

Comment (soltysh) [reply staging/src/k8s.io/code-generator/kube_codegen.sh:602]: > is passing `--openapi-schema ""` valid ?

That's the default, see here: https://github.com/kubernetes/kubernetes/blob/531726ee654ec318b6f4be53a64700e2577629fe/staging/src/k8s.io/code-generator/cmd/applyconfiguration-gen/args/args.go#L75-L76

**Pull request merged**